packages/cx-layout/src/components/Header/index.ts

```diff
@@ -1,2 +1,17 @@
-export * from './HeaderClean';
-export * from './token';
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { HeaderClean, asHeaderToken } from './HeaderClean';
```

Comment:
Why did you cahnge this to explicit exports?

Reply:
I remember we've agreed to only use explicit exports a few weeks ago when SonarQube started marking `export *` as code smell. Should I go back to the previous syntax?

Reply:
oh i forgot about that entirely.  thanks.